samples/synchronization/README.rst

```diff
@@ -30,16 +30,44 @@ Sample Output
 
 .. code-block:: console
 
-   threadA: Hello World!
-   threadB: Hello World!
-   threadA: Hello World!
-   threadB: Hello World!
-   threadA: Hello World!
-   threadB: Hello World!
-   threadA: Hello World!
-   threadB: Hello World!
-   threadA: Hello World!
-   threadB: Hello World!
+   thread_a: Hello World from cpu 0 on qemu_x86!
+   thread_b: Hello World from cpu 0 on qemu_x86!
+   thread_a: Hello World from cpu 0 on qemu_x86!
+   thread_b: Hello World from cpu 0 on qemu_x86!
+   thread_a: Hello World from cpu 0 on qemu_x86!
+   thread_b: Hello World from cpu 0 on qemu_x86!
+   thread_a: Hello World from cpu 0 on qemu_x86!
+   thread_b: Hello World from cpu 0 on qemu_x86!
+   thread_a: Hello World from cpu 0 on qemu_x86!
+   thread_b: Hello World from cpu 0 on qemu_x86!
+
+   <repeats endlessly>
+
+And, if the board was selected qemu_x86_64, this demonstration would 
+show the following results.
```

Comment:
It would be nice to clarify that the purpose of the first example is to test non-SMP, and the second is to test SMP.